zad1/js/scripts.js

```diff
@@ -0,0 +1,21 @@
+$(document).ready(function(){
```

Comment:
idea czekania na zaladowanie drzewa DOM poprzez jQuerowa funkcje `$(document).ready()` jest taka, ze to, co ma sie wykonac po zaladowaniu DOM jest cale wewnatrz tej funkcji. Teraz masz w niej tylko console.log, a reszte poza nia